----------------------------------------------------------------
Mon Nov 09 11:19:32 CET 2020:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance a816c00e-0175-ac84-9676-000002156d50 
on database directory C:\Users\m_ej\Desktop\ContactsApp-2\database with class loader jdk.internal.loader.ClassLoaders$AppClassLoader@5387f9e0 
Loaded from file:/C:/Users/m_ej/.m2/repository/org/apache/derby/derby/10.14.2.0/derby-10.14.2.0.jar
java.vendor=Oracle Corporation
java.runtime.version=14.0.2+12-46
user.dir=C:\Users\m_ej\Desktop\ContactsApp-2
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Mon Nov 09 11:23:55 CET 2020:
Shutting down instance a816c00e-0175-ac84-9676-000002156d50 on database directory C:\Users\m_ej\Desktop\ContactsApp-2\database with class loader jdk.internal.loader.ClassLoaders$AppClassLoader@5387f9e0 
Mon Nov 09 11:23:55 CET 2020 Thread[Thread-20,5,main] Cleanup action starting
ERROR XBM0I: Directory database cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:832)
Cleanup action completed
Mon Nov 09 11:23:55 CET 2020 Thread[Thread-20,5,main] Cleanup action starting
ERROR XBM0I: Directory database cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:832)
Cleanup action completed
----------------------------------------------------------------
Mon Nov 09 11:23:56 CET 2020:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance 61228105-0175-ac84-9676-000002156d50 
on database directory C:\Users\m_ej\Desktop\ContactsApp-2\database with class loader jdk.internal.loader.ClassLoaders$AppClassLoader@5387f9e0 
Loaded from file:/C:/Users/m_ej/.m2/repository/org/apache/derby/derby/10.14.2.0/derby-10.14.2.0.jar
java.vendor=Oracle Corporation
java.runtime.version=14.0.2+12-46
user.dir=C:\Users\m_ej\Desktop\ContactsApp-2
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Mon Nov 09 11:26:53 CET 2020:
Shutting down instance 61228105-0175-ac84-9676-000002156d50 on database directory C:\Users\m_ej\Desktop\ContactsApp-2\database with class loader jdk.internal.loader.ClassLoaders$AppClassLoader@5387f9e0 
Mon Nov 09 11:26:53 CET 2020 Thread[Thread-36,5,main] Cleanup action starting
ERROR XBM0I: Directory database cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:832)
Cleanup action completed
Mon Nov 09 11:26:53 CET 2020 Thread[Thread-36,5,main] Cleanup action starting
ERROR XBM0I: Directory database cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:832)
Cleanup action completed
----------------------------------------------------------------
Mon Nov 09 11:26:53 CET 2020:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance a4420111-0175-ac84-9676-000002156d50 
on database directory C:\Users\m_ej\Desktop\ContactsApp-2\database with class loader jdk.internal.loader.ClassLoaders$AppClassLoader@5387f9e0 
Loaded from file:/C:/Users/m_ej/.m2/repository/org/apache/derby/derby/10.14.2.0/derby-10.14.2.0.jar
java.vendor=Oracle Corporation
java.runtime.version=14.0.2+12-46
user.dir=C:\Users\m_ej\Desktop\ContactsApp-2
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Mon Nov 09 11:30:19 CET 2020:
Shutting down instance a4420111-0175-ac84-9676-000002156d50 on database directory C:\Users\m_ej\Desktop\ContactsApp-2\database with class loader jdk.internal.loader.ClassLoaders$AppClassLoader@5387f9e0 
Mon Nov 09 11:30:20 CET 2020 Thread[Thread-52,5,main] Cleanup action starting
ERROR XBM0I: Directory database cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:832)
Cleanup action completed
Mon Nov 09 11:30:20 CET 2020 Thread[Thread-52,5,main] Cleanup action starting
ERROR XBM0I: Directory database cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:832)
Cleanup action completed
----------------------------------------------------------------
Mon Nov 09 11:30:20 CET 2020:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance a8a40121-0175-ac84-9676-000002156d50 
on database directory C:\Users\m_ej\Desktop\ContactsApp-2\database with class loader jdk.internal.loader.ClassLoaders$AppClassLoader@5387f9e0 
Loaded from file:/C:/Users/m_ej/.m2/repository/org/apache/derby/derby/10.14.2.0/derby-10.14.2.0.jar
java.vendor=Oracle Corporation
java.runtime.version=14.0.2+12-46
user.dir=C:\Users\m_ej\Desktop\ContactsApp-2
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Mon Nov 09 11:32:37 CET 2020:
Shutting down instance a8a40121-0175-ac84-9676-000002156d50 on database directory C:\Users\m_ej\Desktop\ContactsApp-2\database with class loader jdk.internal.loader.ClassLoaders$AppClassLoader@5387f9e0 
Mon Nov 09 11:32:37 CET 2020 Thread[Thread-68,5,main] Cleanup action starting
ERROR XBM0I: Directory database cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:832)
Cleanup action completed
Mon Nov 09 11:32:37 CET 2020 Thread[Thread-68,5,main] Cleanup action starting
ERROR XBM0I: Directory database cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:832)
Cleanup action completed
----------------------------------------------------------------
Mon Nov 09 11:32:38 CET 2020:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance 2faf0139-0175-ac84-9676-000002156d50 
on database directory C:\Users\m_ej\Desktop\ContactsApp-2\database with class loader jdk.internal.loader.ClassLoaders$AppClassLoader@5387f9e0 
Loaded from file:/C:/Users/m_ej/.m2/repository/org/apache/derby/derby/10.14.2.0/derby-10.14.2.0.jar
java.vendor=Oracle Corporation
java.runtime.version=14.0.2+12-46
user.dir=C:\Users\m_ej\Desktop\ContactsApp-2
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
